price_list.go

```diff
@@ -0,0 +1,55 @@
+package shopify
+
+import (
+	"fmt"
+)
+
+// PriceList represents a price list.
+type PriceList struct {
```

Comment:
should this be called PriceList?

Reply:
Isn't that what it's called? 😕

Reply:
sorry I wasn't clear, I think this is a price and not a price list looking at the struct

Reply:
No, it's a price list - see the docs: https://shopify.dev/api/admin-graphql/2021-04/objects/PriceList#fields-PriceList-OBJECT

Reply:
I see so the actual price list has more fields but they are not necessary here